locales/zh_CN/zh_CN.json

```diff
@@ -415,6 +415,7 @@
     "Papua New Guinea": "巴布亚新几内亚",
     "Paraguay": "巴拉圭",
     "Password": "密码",
+    "Password Confirmation": "密码确认",
```

Comment:
Where is this translation used? No such key found in source files.

Reply:
just remove it, please.

Reply:
sorry , this line added by me , it comes from breeze register.blade.php